user-interface/src/case-detail/CaseDetailScreen.test.tsx

```diff
@@ -9,12 +9,14 @@ import { CaseDetail } from '@common/cams/cases';
 import { Debtor, DebtorAttorney } from '@common/cams/parties';
 
 const caseId = '101-23-12345';
-const brianWilsonName = 'Brian Wilson';
-const carlWilsonName = 'Carl Wilson';
-const trialAttorneyRole = 'Trial Attorney';
+
+const brianWilson = { id: '001', name: 'Brian Wilson' };
```

Comment:
Moar mock attorney definitions...

Reply:
More?

Reply:
Yes. Moar.

Reply:
"Trial Attorney" is hard coded in the JSX for BasicInfo panel.